functions/HttpTriggerAPIUsers/function.json

```diff
@@ -9,7 +9,8 @@
       "methods": [
         "get",
         "post"
-      ]
+      ],
+      "route":"users"
```

Comment:
good point! I will go ahead and add the route to my endpoint as well

Reply:
I believe Jenny, Ahmed and were able to rename the endpoints / routes.